modules/common/security/networking.nix

```diff
@@ -0,0 +1,188 @@
+# Copyright 2024-2025 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  ...
+}:
+let
+  cfg = config.ghaf.security.sysctl.network;
+in
+{
+  ## Options to enable IP security
+  options.ghaf.security.sysctl.network = {
+    disable-all = lib.mkOption {
```

Comment:
Why not the typical `enable` flag that defaults to true?

Reply:
Thanks. Done.